src/App.js

```diff
@@ -7,9 +8,9 @@ const App = () => {
     <div id={styles.app}>
       <Router>
         <Switch>
-          {/* <Route path="/path">
-            <Component />
-          </Route> */}
+          <Route path="/path">
+            <Leetcode />
```

Comment:
Can you change this to the original App.js so we don't get too many merge conflicts :))